banbot.js

```diff
@@ -317,7 +315,57 @@ bot.on('message', async ( message ) => {
             else {
                 if ( !leftServer ) {
                     target.user.send( 'You\'ve been banned in the CPL Discord server. If you believe this to be an error, please send an email to civplayersleagues@gmail.com' )
-                    target.ban( 'Tier 6 major' )
+                    target.ban( 'Tier 4 major' )
+                }
+                else mongo.banDue( id )
+            }
+        })
+    }
+    else if ( content.startsWith( '.extreme' ) ) {
+        message.delete()
+        let target = message.mentions.members.array().shift()
+        let id = target ? target.id : null
+        if ( !target ) {
+            if ( content.split( ' ' ).length > 1 ) {
+                target = content.split(' ')[1]
+                id  = target.replace( /\D*/g, '' )
+                var leftServer = true
+            }
+        }
+        if ( !target || !id ) {
+            message.channel.send( extreme ).then( msg => { msg.delete( 30000 ) } )
+            return
+        }
+        let reason = content.split(' ').slice(2).join(' ')
+
+        let player = await mongo.extreme( id )
+        let msg = '\n**[ TIER ' + player.tier + ' EXTREME INFRACTION ]**'
+        if ( reason.length > 0 )
+            msg += '\n**REASON:** ' + reason + '.'
+        if ( player.tier == 1 )
+            msg += '\n**RESULT:** 30 day suspension.'
+        else if ( player.tier >= 2 )
+            msg += '\n**RESULT:** ' + target.displayName + ' banned from server. '
+        if ( !leftServer && target.roles.has( suspendedId ) && player.tier < 4 )
+            msg += ' This is in addition to your current suspension.'
```

Comment:
`player.tier < 4` feels like a copy/paste error here, since there's only 2 extreme tiers. I think these 2 lines are simply unnecessary for the extreme tier, since the 2nd one is a ban

Reply:
fixed in new push